lewis/adapters/stream.py

```diff
@@ -86,7 +84,44 @@ def handle_accept(self):
             StreamHandler(sock, self.target)
 
 
-class Cmd(object):
+class CanNotProcessException(Exception):
+    pass
+
+
+class BaseCommand(object):
+    member = None
+    argument_mappings = None
+    return_mapping = None
```

Comment:
Well in this case I'd say we take the easy way out and leave it as it is until somebody tells us "I have no idea what you're talking about".